templates/registration/logged_out.html

```diff
@@ -0,0 +1,7 @@
+{% extends "base.html" %}
+{% load crispy_forms_tags %}
+
+{% block content %}
+  <p>Logged out!</p>
+  <a href="{% url 'login'%}">Click here to login again.</a>
```

Comment:
Consider adding some spacing between 'login' and the closing quotation mark for better readability.